### ISSUE TITLE
`is_model_meta_subclass` doesn't detect subclasses of `Model` subclasses

### ISSUE THREAD

**Issue by supervacuo** [PyCQA/pylint-django#2]
Define the following models:

```
class FooModel(models.Model):
...
    class Meta:
        abstract = True

class BarModel(FooModel):
    ...
    class Meta:
        ....
```

Then, enable `pylint-django` and run `pylint` on a module including these models.

Expected result: `C1001`, `W0232`, `R0903` disabled for `BarModel.Meta`.

Actual result: `pylint` messages as if `pylint-django` weren't included (but messages for `Meta` on models inheriting directly from `models.Model` are suppressed as expected)
